src/blog/2023/01/getting-started-with-node-red.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting Started with Node-RED
+subtitle: Node-RED is one of the easiest ways to program ever created but everyone needs a little help
+description: In this article we are going to explain the first things we think you need to know to get started with Node-RED
+date: 2023-01-23
+authors: ["rob-marcer"]
+---
+
+
+Node-RED is a visual programming tool for working with IoT devices and web services. It allows users to create flows using a drag-and-drop interface, making it easy to connect different nodes together to build powerful automations.
+<!--more-->
+In this blog post, we'll take a look at how to get started with Node-RED and create some basic flows. Additionally, we will explore the use of the palette manager, a powerful feature that allows users to install and manage additional nodes for Node-RED.
+
+### Installing Node-RED
+
+First, you'll need to install Node-RED on your machine. The easiest way to do this is by using npm (Node Package Manager), which comes with Node.js. To install Node-RED, open up your terminal and type the following command:
+```
+npm install -g node-red
+```
+Once Node-RED is installed, you can start it by running the following command:
+```
+node-red
+```
+
+This will start the Node-RED server and open up the editor in your web browser. You can also specify a different port or a settings file if you need to.
```

Comment:
```suggestion
This will start the Node-RED server and open up the [editor in your web browser](http://localhost:1880). You can also specify a different port or a settings file if you need to.
```